assets/aml-benchmark/scripts/custom_dataset_preprocessors/quac_textgen_babel.py

```diff
@@ -0,0 +1,115 @@
+# ---------------------------------------------------------
```

Comment:
Why is there `babel` in the name of this file?

Reply:
This preprocessor logic is taken from babel. The preprocessor and creation of few shot examples for this dataset are different on different benchmark platforms.